### ISSUE TITLE
Keyboard navigation on q-table does not work

### ISSUE THREAD

**Issue by mirkobrauns** [quasarframework/quasar#8539]
**Describe the bug**
The Keyboard navigation on q-table does not work, because the computed props `computedRowsNumber` and `computedRowsSee` cannot be accessed. See (https://next.quasar.dev/vue-components/table#keyboard-navigation)

**Codepen/jsFiddle/Codesandbox (required)**
See Codepen on [QTable Keyboard navigation example](https://next.quasar.dev/vue-components/table#keyboard-navigation)

**To Reproduce**
1. Go to https://next.quasar.dev/vue-components/table#keyboard-navigation
2. Click on the checkbox of the first line.
3. Press arrow down key
4. Nothing happens and in the browser console `TypeError: undefined is not an object (evaluating 'r.length')` will logged.

**Expected behavior**
After pressing the arrow down key, the next table row must be selected.

**Platform (please complete the following information):**
Quasar Version: v2.0.0-beta.7
@quasar/app Version: v3.0.0-beta7
Quasar mode:
  [x] SPA
  [ ] SSR
  [ ] PWA
  [ ] Electron
  [ ] Cordova
  [ ] Capacitor
  [ ] BEX
Tested on:
  [x] SPA
  [ ] SSR
  [ ] PWA
  [ ] Electron
  [ ] Cordova
  [ ] Capacitor
  [ ] BEX
OS: maxOS Big Sur V11.2.1, Win 10
Browsers: Safari V14.0.3, Firefox: 87.0b2, Chrome 88.0.4324.181

**Comment by rstoenescu** [comment]
Thanks for reporting. Fix will be available in next beta

**Comment by JayFett** [comment]
This issue still appears to be outstanding and reproducible as described (after rc1 released)
